code/controllers/subsystem/processing/goap.dm

```diff
@@ -0,0 +1,20 @@
+PROCESSING_SUBSYSTEM_DEF(goap)
```

Comment:
No. This is separate from NPC Pool and will not be merged with it.

GOAP isn't only for AI. It can be used for any system in which a lowest cost solution must be discovered.